fbw-a32nx/src/wasm/fadec_a320/src/EngineControl.h

```diff
@@ -733,6 +731,9 @@ class EngineControl {
     double rightPump1 = simVars->getPump(3);
     double rightPump2 = simVars->getPump(6);
 
+    base_arinc_429 apuNpercentWord = Arinc429Utils::fromSimVar(simVars->getAPUrpmPercent());
+    double apuNpercent = Arinc429Utils::valueOr(apuNpercentWord, 0);
```

Comment:
It would be more suitable to use `A32NX_APU_N_RAW` here. ARINC values are transmitted between computers and will not be correct in case of sensor/computer failures. The fuel consumption should not rely on those.

Reply:
Fair point